### PR TITLE
Added component/insights label openshift/console labels

### PR DESCRIPTION
**Pull request by bond95**
It's relates to new plugin that was added to openshift console: https://github.com/openshift/console/pull/6660

### CODE REVIEW

File: core-services/prow/02_config/_labels.yaml [hunk 1]
```diff
@@ -604,6 +604,11 @@ repos:
         name: docs-approved
         target: both
         addedBy: label
+      - color: 00D8FF
+        description: Related to insights plugin
+        name: component/insights
```
Comment (spadgett) [inline core-services/prow/02_config/_labels.yaml:609]: The other component labels are alphabetized. Can we move this after kubevirt above?

**Review by spadgett — COMMENTED** [review]
Thanks, just one comment

File: core-services/prow/02_config/_labels.yaml (hunk 1 above)
Comment (bond95) [reply core-services/prow/02_config/_labels.yaml]: Done

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
@bond95 Hi Bohdan! Long time no see :wave:

**Comment by bond95** [comment]
@petr-muller Hey 😃

**Pull request merged**